.github/workflows/deploy-nft-contract.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy NFT contract
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - fc-community-contracts/src/FactChainNFT.sol
+      - fc-community-contracts/src/utils/Ownable.sol
+      - fc-community-contracts/src/ERC721/**
+jobs:
+  deploy-contract:
+    runs-on: ubuntu-latest
+    env:
+      CONTRACT_NAME: FactChainNFT
+      CHAIN_ID: 11155111
+      ETH_RPC_URL: ${{ secrets.RPC_ETH_SEPOLIA_HTTPS }}
+      PRIVATE_KEY: ${{ secrets.DEPLOYER_PK }}
+      ETHERSCAN_URL: https://sepolia.etherscan.io
+      ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_SEPOLIA_API_KEY }}
+      OWNER_ADDRESS: ${{ vars.OWNER_ADDRESS }}
+    permissions: write-all
+    defaults:
+      run:
+        working-directory: ./fc-community-contracts
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+
+      - name: Build contract
+        run: forge build
+
+      - name: Deploy contract
+        id: deploy
+        run: |
+          createOutput=`forge create "src/${{ env.CONTRACT_NAME }}:${{ env.CONTRACT_NAME }} --verify --json --rpc-url=$ETH_RPC_URL --private-key=$PRIVATE_KEY --constructor-args $OWNER_ADDRESS`
```

Comment:
Once you have [fixed the constructor](https://github.com/factchain/factchain-community/pull/30/files#diff-c70b1ef67d14901ef1793d4d9f6b6b94225bb6b6c041892b11e23d5f1fb045dcR16) can you change the `--constructor-args` being passed here?